spec/system/candidate_interface/candidate_entering_contact_details_spec.rb

```diff
@@ -65,6 +66,10 @@ def when_i_click_on_contact_details
     click_link t('page_titles.contact_details')
   end
 
+  def and_i_click_on_change_my_phone_number
+    all('.govuk-summary-list__actions')[0].click_link 'Change'
```

Comment:
There's hidden alt text you can use instead:

```suggestion
    click_link 'Change phone number'
```